config/default/forms/app/delivery.xml

```diff
@@ -1433,14 +1433,14 @@ selected(../convulsion, 'yes'), 'yes', 'no')"/>
       <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/baby_name" type="string" required="true()"/>
       <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/baby_sex" type="select1" required="true()"/>
       <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/birth_weight_know" type="select1" required="true()"/>
-      <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/birth_weight" type="decimal" relevant="selected(../birth_weight_know, 'yes')" constraint=". &gt;= 1500 and . &lt;= 5000" required="true()" jr:constraintMsg="jr:itext('/delivery/babys_condition/baby_repeat/baby_details/birth_weight:jr:constraintMsg')"/>
+      <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/birth_weight" type="decimal" constraint=". &gt;= 1500 and . &lt;= 5000" required="true()" jr:constraintMsg="jr:itext('/delivery/babys_condition/baby_repeat/baby_details/birth_weight:jr:constraintMsg')" relevant="selected(../birth_weight_know, 'yes')"/>
       <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/birth_length_know" type="select1" required="true()"/>
-      <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/birth_length" type="decimal" relevant="selected(../birth_length_know, 'yes')" constraint=". &gt;= 35.6 and . &lt;= 70" required="true()" jr:constraintMsg="jr:itext('/delivery/babys_condition/baby_repeat/baby_details/birth_length:jr:constraintMsg')"/>
+      <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/birth_length" type="decimal" constraint=". &gt;= 35.6 and . &lt;= 70" required="true()" jr:constraintMsg="jr:itext('/delivery/babys_condition/baby_repeat/baby_details/birth_length:jr:constraintMsg')" relevant="selected(../birth_length_know, 'yes')"/>
       <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/vaccines_received" type="select1" required="true()"/>
-      <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/imm_counsel_note" readonly="true()" type="string" relevant="selected(../vaccines_received, 'none')" required="true()"/>
+      <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/imm_counsel_note" readonly="true()" type="string" relevant="selected(../vaccines_received, 'none')"/>
       <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/breatfeeding" type="select1" required="true()"/>
       <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/breastfed_within_1_hour" type="select1" required="true()"/>
-      <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/baby_danger_sign_note" readonly="true()" type="string" required="true()"/>
+      <bind nodeset="/delivery/babys_condition/baby_repeat/baby_details/baby_danger_sign_note" readonly="true()" type="string"/>
```

Comment:
This was the change that I had to regenerate this form to pick up.  You can no longer submit a form if you have a required `note` field (even though there is no way to fill out a value for the field).